apps/server/src/middlware/authorization/authentication.ts

```diff
@@ -82,3 +83,17 @@ export const validateRefreshTokenMiddleWare = async (
       : JWT_INVALID_STATUS_CODE,
   })
 }
+
+export const extractJwtMiddleware = (
+  req: JwtPayload,
+  res: Response,
+  next: NextFunction
+) => {
+  const token = req.headers.authorization?.split(' ')[1]
+  if (token) {
+    req.refreshToken = token
+    next()
+  } else {
+    return res.status(401).json({ message: 'Unauthorized' })
+  }
```

Comment:
I need clarity on this

Reply:
I also believe this portion of the code no longer exists in the codebase & must have found its way here from the little confusion i had while resolving conflict & having multiple commits